app/Console/Command/ProcessDuplicatePlaces.php

```diff
@@ -45,14 +46,16 @@ public function __construct(
         CanonicalService $canonicalService,
         EventBus $eventBus,
         DocumentEventFactory $placeEventFactory,
-        EventRelationsRepository $eventRelationsRepository
+        EventRelationsRepository $eventRelationsRepository,
+        LoggerInterface $logger
     ) {
         $this->duplicatePlaceRepository = $duplicatePlaceRepository;
         $this->duplicatePlaceRemovedFromClusterRepository = $duplicatePlaceRemovedFromClusterRepository;
         $this->canonicalService = $canonicalService;
         $this->eventBus = $eventBus;
         $this->placeEventFactory = $placeEventFactory;
         $this->eventRelationsRepository = $eventRelationsRepository;
+        $this->logger = $logger;
```

Comment:
Why do we need a logger in the context of a CLI script?

Reply:
We normally use the standard output and use `$output` from the `execute` method. I don't think there is a specific file to log CLI command output. This would be the first CLI command that uses `LoggerInterface`.

Reply:
Well there is one, `LoggerName::forService('fetching-movies', 'kinepolis')` but that is setup as a scheduled task with a dedicated log file.